### ISSUE TITLE
Tag BDB 1.7 parts

### ISSUE THREAD

**Issue by DeltaDizzy** [KerbalKore/CWPT#32]
Should we wait until new parts stop coming out? What about the tech rebalance Zorg is doing?

**Comment by KerbalKore** [comment]
I would agree with that. We should focus on tagging already added parts and stock parts.
